digiwf-email-integration-starter/src/main/java/io/muenchendigital/digiwf/email/integration/configuration/MailAutoConfiguration.java

```diff
@@ -0,0 +1,57 @@
+package io.muenchendigital.digiwf.email.integration.configuration;
+
+import io.muenchendigital.digiwf.email.integration.domain.service.MailingService;
+import io.muenchendigital.digiwf.s3.integration.client.configuration.S3IntegrationClientAutoConfiguration;
+import io.muenchendigital.digiwf.s3.integration.client.repository.DocumentStorageFileRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.autoconfigure.AutoConfigureAfter;
+import org.springframework.boot.autoconfigure.mail.MailProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.mail.javamail.JavaMailSender;
+
+import javax.mail.MessagingException;
+
+@Configuration
+@RequiredArgsConstructor
+@AutoConfigureAfter({ S3IntegrationClientAutoConfiguration.class })
+//TODO: can be switched when the S3 library updated its config
+//@ComponentScan(basePackages = {"io.muenchendigital.digiwf.email.integration"})
+@ComponentScan(basePackages = {"io.muenchendigital.digiwf.email.integration","io.muenchendigital.digiwf.s3.integration.client"})
+@EnableConfigurationProperties({MailProperties.class, CustomMailProperties.class})
+public class MailAutoConfiguration {
+
+    private final MailProperties mailProperties;
+    private final CustomMailProperties customMailProperties;
+    private final MailConfiguration mailConfiguration;
+
+    /**
+     * Configures the {@link JavaMailSender}
+     *
+     * @return configured JavaMailSender
+     */
+    @Bean
+    public JavaMailSender getJavaMailSender() throws MessagingException {
```

Comment:
@ConditionalOnMissingBean
Would always add it to basic Beans.

Reply:
Done, changed it also for ```public MailingService getMailingService```